node/tests/GlideClient.test.ts

```diff
@@ -566,7 +571,7 @@ describe("GlideClient", () => {
     it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
         "copy with DB test_%p",
         async (protocol) => {
-            if (cluster.checkIfServerVersionLessThan("6.2.0")) return;
+            if (cluster.checkIfServerVersionLessThan("6.3.0")) return;
```

Comment:
I think it is better to fix `checkIfServerVersionLessThan` and keep comparing with `6.2.0` according to the doc.

Reply:
what is the fix you offer?

Reply:
```diff
-    return lt(this.version, minVersion);
+    return gt(minVersion, this.version);
```

Reply:
Or i miss something, or its the same.
v6.2.14
```
cluster.checkIfServerVersionLessThan("6.2.0")
lt("6.2.14", "6.2.0"); -> false
gt("6.2.0", "6.2.14"); -> false
```
Im not sure its matter a lot. we want to check that a version is bigger than 6.2, not 6.2.0. so 6.3 is the right one to use.
In other version, we usually check to see if a feature is already implemented so 7.0.0 is fine, in this case this is our bottom limit, so we just want to see that its 6.2.* and bellow.
I saw that you sometime add checks for lower versions (6.0), Its not needed, I actually wanted to remove it since its extra logic with no need, its noise.
But i respect your work so added it back.
But - we are not going to complicate further for versions we not support.

Reply:
```
lt("6.2.14", "6.2.0"); -> false
gt("6.2.0", "6.2.14"); -> false
```
just because tests are running on `6.2.14` - a lucky coinsidence. Once you start it on 6.2 it will fail.
Version 6.3 isn't mentioned in command docs, we cannot use it in version checks.

Reply:
6.3 is used just as a supremum of 6.2. 
We could use 6.2.999999 as well, but it's not a supremum per se, and there's no reason to do so.
Once a version is lower than 6.3.0, it has to be 6.2 or lower, which is what we care about.

Reply:
We can do the same for any other version check, but it is a bad approach. Compare version (or anything) with a magic number nowhere mentioned and not related to a feature - a road to a failure which could be difficult to resolve.

Reply:
I don't think that an upper bound is a magic number, but what is the other option? Does semver allow comparing major.minor and avoiding the patch

Reply:
I'll find a solution